halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/azure/AzureAddAccountCommand.java

```diff
@@ -89,8 +94,22 @@ protected String getProviderName() {
   )
   private String packerStorageAccount;
 
+  @Parameter(
+      names = "--regions",
+      variableArity = true,
+      description = AzureCommandProperties.REGIONS_DESCRIPTION
+  )
+  private List<String> regions = new ArrayList<>();
+
+
   @Override
   protected Account buildAccount(String accountName) {
+    // Add default regions if user not specified
```

Comment:
Is there a reason not to just specify the defaults directly in the initializer for the `regions` field?

Reply:
@lwander Which style is preferable?

Reply:
Definitely preferred to declare these on line 102 above, rather than rely on the empty list as a default

Reply:
Apologize for that, I'm not quite familiar with Java so after simple searching I chose to add the default value after the initializer. Now I updated the default value in initializer as
```
private List<String> regions = Arrays.asList("westus", "eastus");
``` 
Is that OK?